examples/test/Menu_test.js

```diff
@@ -0,0 +1,158 @@
+// (C) 2007-2018 GoodData Corporation
+import { Selector } from 'testcafe';
+import { config } from './utils/config';
+import { loginUsingLoginForm, waitForPivotTableStopLoading } from './utils/helpers';
+
+const totalValues = {
+    sum: ['Sum', '', '', '$1,566,007', '$150,709'],
+    max: ['Max', '', '', '$101,055', '$25,140'],
+    empty: '–'
+};
+
+const turnOnMenu = async (t) => {
+    await t.click(Selector('.s-total-preset-aggregations'));
+    await waitForPivotTableStopLoading(t);
+};
+
+const getMeasureCell = (column) => {
+    return Selector(`[col-id="a_2009_1-a_2071_1-m_${column}"] .s-pivot-table-column-header`);
```

Comment:
Is there an `s-` based selector instead of the attribute one? If not, would it be hard to add one?

Reply:
I decided to use this attribute selector because: 
- ag-grid does not easily expose a way to add s- classes that we would need in this specific case (header row that is measure and is not a group)
- attribute selectors are used in standard pivot table tests